lib/logstash/filters/ruby.rb

```diff
@@ -13,6 +13,15 @@
 #       } 
 #     } 
 #
+# If you need to create additional events, it cannot be done as in other filters where you would use `yield`,
+# you must use a specific syntax `new_event_block.call(event)` like in this example duplicating the input event
```

Comment:
What about having a helper method `yield_event` that folks can call? `yield_event(whatever)` thoughts?